test/unit/accord/utils/Gen.java

```diff
@@ -87,6 +89,22 @@ default Gen<A> filter(Predicate<A> fn)
         };
     }
 
+    default Gen<A> filter(int maxAttempts, A defaultValue, Predicate<A> fn)
```

Comment:
a lot of changes to these classes are not *needed* for this patch, but my thinking is to try to make in-sync with accord so the next rebase isn't a pain